manifests/params.pp

```diff
@@ -3,6 +3,7 @@
 # This class is meant to be called from consul
 # It sets variables according to platform
 #
+# @api private
```

Comment:
no assert here?

Reply:
ah that comment was a leftover. we cannot mark the params class as private. I removed the comment.